ra/ra.go

```diff
@@ -292,9 +289,9 @@ func (ra *RegistrationAuthorityImpl) NewRegistration(ctx context.Context, init c
 	// Store the authorization object, then return it
 	reg, err = ra.SA.NewRegistration(ctx, reg)
 	if err != nil {
-		// InternalServerError since the user-data was validated before being
+		// berrors.InternalServer since the user-data was validated before being
 		// passed to the SA.
-		err = core.InternalServerError(err.Error())
+		err = berrors.InternalServerError("ra.NewRegistration: %s", err)
```

Comment:
Internal Server errors are never returned to the user, see: https://github.com/letsencrypt/boulder/blob/8a1adbdc9ab5bea51a6bb676cac76806aab3b8d6/core/util.go#L137